homework/yaryna/find_most_frequent_hw_4.py

```diff
@@ -0,0 +1,21 @@
+string = "Mom! Mom! Are you sleeping?! you!!"
+
+string = string.lower()
+string = string.replace("-"," ")
+string = string.replace(",","")
+string = string.replace("!","")
+string = string.replace("?","")
+string = string.replace(".","")
```

Comment:
you can use chain of replace() calls `string = string.replace(".","").replace("?","")` etc.